ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -60,10 +64,30 @@ images:
     COPY . .
   from: src
   to: assisted-service-scripts
+- dockerfile_path: deploy/olm-catalog/bundle.Dockerfile
+  to: assisted-service-operator-bundle
 - dockerfile_literal: |
-    FROM base
-  from: ci-index
-  to: assisted-service-bundle
+    FROM quay.io/operator-framework/upstream-opm-builder:latest AS builder
+
+    ARG BUNDLE_IMGS=quay.io/edge-infrastructure/assisted-service-operator-bundle:latest
```

Comment:
it is a good idea to make sure the index is referencing the bundle with a fixed tag (like digest value) and not ``latest``, otherwise each push to the ``latest`` tag will make old indexes reference a newer bundle image

for the job scope it's probably not an issue. once we mirror it to quay.io, our scope is more global and problems might arise

Reply:
>it is a good idea to make sure the index is referencing the bundle with a fixed tag (like digest value) and not latest, otherwise each push to the latest tag will make old indexes reference a newer bundle image

yep we're aware. It's still better than what we have now. We're trying to come up with a better solution